app/controllers/questions_controller.rb

```diff
@@ -1,27 +1,89 @@
 class QuestionsController < ApplicationController
   def show
-    @user = current_user
-    user_id = @user.id
 
+    @user = current_user
```

Comment:
Tu som mala problem s riadkom: @feedbacks = @question.feedbacks.visible.includes(:user)

Pretoze takto to zbehne, ked mam @user, ale ked som namiesto usera pouzila current_user, tak to neslo, nechapem preco.